backend/app/cmd/server.go

```diff
@@ -72,6 +72,7 @@ type ServerCommand struct {
 	WebRoot          string        `long:"web-root" env:"REMARK_WEB_ROOT" default:"./web" description:"web root directory"`
 	UpdateLimit      float64       `long:"update-limit" env:"UPDATE_LIMIT" default:"0.5" description:"updates/sec limit"`
 	RestrictedWords  []string      `long:"restricted-words" env:"RESTRICTED_WORDS" description:"words prohibited to use in comments" env-delim:","`
+	RestrictedNames  []string      `long:"restricted-names" env:"RESTRICTED_NAMES" description:"names prohibited to use by user" env-delim:","`
```

Comment:
Should we mention "prohibited to use by the anonymous users"? AFAIK we fetch whatever user has in GitHub and in other places, and it's relevant only for anonymous and email users but latter are out of the scope. Otherwise, looks good (aside from many unrelated changes like go.sum).

Reply:
i think we better keep it more universal. Currently we restrict anon and email only, however it may be a good idea to restrict any auth method with the similar logic. I'm not brave enough to do it right away but this is smth to consider